linux_os/guide/system/software/integrity/crypto/configure_gnutls_tls_crypto_policy/ansible/shared.yml

```diff
@@ -0,0 +1,23 @@
+# platform = Red Hat Enterprise Linux 8
```

Comment:
Is there a reason why not to include `fedora` platform in the remediations? since the rule's `prodtype` has `fedora` in it.

Reply:
the same in bash remediation

Reply:
fixed

Reply:
I'm afraid it needs to be capitalized `Fedora`, see: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/tests/augenrules_default.fail.sh#L3